quipucords/api/source/serializer.py

```diff
@@ -460,9 +429,9 @@ def validate_hosts(hosts):
                 # This is an unexpected case. Allow/log for analysis
                 normalized_hosts.append(host)
                 logging.warning('%s did not match a pattern or produce error',
-                                host_range)
+                                host)
         if len(host_errors) is 0:
-            return normalized_hosts
+            return json.dumps(normalized_hosts)
```

Comment:
Is there value in dumping to JSON instead of passing the python object?  Since you just end up turning around and doing a `json.loads` in the `create` method.

Reply:
Good point.  Changed.